src/AcceptanceTesting/SpecByExample/SFA.DAS.Payments.AcceptanceTests/Assertions/SubmissionDataLockRules/CompletionDataLockRule.cs

```diff
@@ -12,7 +12,7 @@ public CompletionDataLockRule() : base("completion")
 
         protected override IEnumerable<SubmissionDataLockResult> FilterPeriodStatuses(SubmissionDataLockPeriodResults periodStatuses)
         {
-            return periodStatuses.Matches.Where(m => m.TransactionType == ReferenceDataModels.TransactionType.Completion);
+            return periodStatuses.Matches.Where(m => m.TransactionType == ReferenceDataModels.TransactionType.Completion );
```

Comment:
this and EnglishAndMathsOnProgrammeDataLockRule are more or less identical in their implementation. I would be inclined to pull up the FilterPeriodStatuses method up to the SubmissionDataLockRuleBase class